src/setup/metaMaskDownloader.ts

```diff
@@ -1,11 +1,14 @@
-import * as fs from 'fs';
-import { IncomingMessage } from 'http';
-import { get } from 'https';
-import * as path from 'path';
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
```

Comment:
this can be one liner

```suggestion
/* eslint-disable @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access  */
```